web/src/components/RemoteContent.tsx

```diff
@@ -10,12 +10,8 @@ const SandBox = styled.div<{ centered: boolean; smallText: boolean }>`
     props.smallText ? `clamp(0.55rem, 1.6vh, ${props.theme.fonts.hintFontSize})` : props.theme.fonts.contentFontSize};
   line-height: ${props => props.theme.fonts.contentLineHeight};
 
-  ${props =>
-    props.centered &&
-    css`
-      text-align: center;
-      list-style-position: inside;
-    `}
+  ${props => (props.centered ? 'text-align: center;' : '')}
```

Comment:
Same as above, I don't think it is a good approach to set a attribute to an invalide value

Reply:
Same as above

Reply:
hm i also don't see the benefit in the refactoring. Do you think its better readable? 
Can not start webpack to the if it results in invalid css, check comment

Reply:
I definitely think so, yes. CSS attributes are defined separately which makes it better readable (at least in my opinion). We also don't need this weird `css` helper. However, I did not change this because I wanted to but because it didn't work together with stylelint.